app/src/format.py

```diff
@@ -113,11 +113,6 @@ def _format_to_accordion_group_html(documents: OrderedDict[Document, list[ChunkW
             chunk = chunk_with_score.chunk
             formatted_chunk = _replace_bem_with_link(chunk.content)
 
-            # Adjust markdown for lists so Chainlit renders correctly
-            formatted_chunk = re.sub("^    - ", "- ", formatted_chunk, flags=re.MULTILINE)
-            if formatted_chunk.startswith("- "):
-                formatted_chunk = "\n" + formatted_chunk
```

Comment:
See https://github.com/navapbc/labs-decision-support-tool/pull/73/files#r1745798163. Are you certain this can be removed?

Reply:
You can check by re-ingesting using this PR and looking at the `chunks` table and querying `content LIKE '    -%'` or `content LIKE '-%'`. Here's what my current DB looks like:
<img width="617" alt="image" src="https://github.com/user-attachments/assets/5d5b755a-0b8a-4476-a3be-61043304f505">

Reply:
The output with the updated code looks correct. double clicking on the entry seems to format the results properly for me and the test cases are passing with the lines removed.
<img width="634" alt="Screenshot 2024-09-06 at 12 55 27 PM" src="https://github.com/user-attachments/assets/b9dceba6-b7dc-439d-9401-b7adb4f2c2a4">

Reply:
Right, so if there are chunks that start with `-`, do they render correctly in the UI? I think this code was intended to correct some list rendering issues, i.e., the UI wouldn't render as a list item without a prepending `\n`.

Reply:
I was able to replicate the issue, re-added the code with updates